### PR TITLE
Add solutions and specs for exercises

### PR DESCRIPTION
**Pull request by AnnAllan**

### CODE REVIEW

File: lib/anagrams.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def anagrams(word, dictionary_array)
+  sorted_input = word.split("").sort
```
Comment (remis1889) [inline lib/anagrams.rb]: check if first argument is a `String`

File: spec/anagrams_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'anagrams'
+describe 'anagrams' do
```
Comment (remis1889) [inline spec/anagrams_spec.rb:2]: test argument types

File: lib/stock_picker.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+def stock_picker(prices)
+  profit = 0
```
Comment (remis1889) [inline lib/stock_picker.rb]: raise an error if the argument is not `Array` and if the length of prices array is less than 2

File: spec/stock_picker_spec.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'stock_picker'
+describe 'stock_picker' do
```
Comment (remis1889) [inline spec/stock_picker_spec.rb:2]: test edge cases

File: lib/anagrams.rb (hunk 1 above)
Comment (AnnAllan) [reply lib/anagrams.rb]: Completed.

File: spec/anagrams_spec.rb (hunk 2 above)
Comment (AnnAllan) [reply spec/anagrams_spec.rb:2]: Completed.

File: lib/stock_picker.rb (hunk 3 above)
Comment (AnnAllan) [reply lib/stock_picker.rb]: Completed.

File: spec/stock_picker_spec.rb (hunk 4 above)
Comment (AnnAllan) [reply spec/stock_picker_spec.rb:2]: Are there additional edge cases that I can cover beyond the highest price on the first day and lowest price on the last?  I have added those tests, but just wondering if there are more.
Completed.
